src/portal-components/DetailsPage.tsx

```diff
@@ -170,94 +148,110 @@ export default class DetailsPage extends React.Component<
-      synapseConfigArray.map((el: RowSynapseConfig, index) => {
-        const style: React.CSSProperties = {}
-        const { columnName = '' } = el
-        const isDisabled =
-          queryResultBundle &&
-          !row[mapColumnHeaderToRowIndex[columnName]] &&
-          !el.standalone
-        if (isDisabled) {
-          style.color = '#BBBBBC'
-          style.cursor = 'not-allowed'
-        }
-        const className = `menu-row-button ${
-          isDisabled ? '' : 'SRC-primary-background-color-hover'
-        }`
-        if (el.name === 'ExternalFileHandleLink') {
-          return (
-            <ExternalFileHandleLink
-              className={className}
-              synId={el.props.synId}
-            />
-          )
-        }
+const handleMenuClick = (
+  ref: React.RefObject<HTMLDivElement>,
+  index: number) => {
+  const wrapper = ref.current?.querySelector<HTMLDivElement>(
+    `#${COMPONENT_ID_PREFIX}${index}`,
+  )
+  if (wrapper) {
+    // https://stackoverflow.com/a/49924496
+    const offset = 85
+    const bodyRect = document.body.getBoundingClientRect().top
+    const elementRect = wrapper.getBoundingClientRect().top
+    const elementPosition = elementRect - bodyRect
+    const offsetPosition = elementPosition - offset
+    window.scrollTo({
+      top: offsetPosition,
+      behavior: 'smooth',
+    })
+  } else {
+    console.error('Could not scroll to element with index ', index)
+  }
+}
+const renderMenu = (
+  ref: React.RefObject<HTMLDivElement>,
+  synapseConfigArray?: RowSynapseConfig[],
+  queryResultBundle?: QueryResultBundle
+) => {
+  const mapColumnHeaderToRowIndex: Dictionary<number> = {}
+  let row: string[] = []
+  if (queryResultBundle) {
+    queryResultBundle.queryResult.queryResults.headers.forEach(
+      (el, index) => {
+        mapColumnHeaderToRowIndex[el.name] = index
+      },
+    )
+    row = queryResultBundle.queryResult.queryResults.rows[0].values
+  }
+  return (
+    synapseConfigArray &&
+    synapseConfigArray.map((el: RowSynapseConfig, index) => {
+      const style: React.CSSProperties = {}
+      const { columnName = '' } = el
+      const isDisabled =
+        queryResultBundle &&
+        !row[mapColumnHeaderToRowIndex[columnName]] &&
+        !el.standalone
+      if (isDisabled) {
+        style.color = '#BBBBBC'
+        style.cursor = 'not-allowed'
+      }
+      const className = `menu-row-button ${
+        isDisabled ? '' : 'SRC-primary-background-color-hover'
+      }`
+      if (el.name === 'ExternalFileHandleLink') {
         return (
-          <button
-            style={style}
-            key={JSON.stringify(el)}
-            onClick={isDisabled ? undefined : () => this.handleMenuClick(index)}
+          <ExternalFileHandleLink
             className={className}
-          >
-            {el.title}
-          </button>
+            synId={el.props.synId}
+          />
         )
-      })
-    )
-  }
+      }
+      if (!el.title) {
```

Comment:
This is a conditional check that I added. If the config has no title, then do not include it in the left-hand side menu.